codeship-steps.yml

```diff
@@ -6,10 +6,26 @@
   steps:
   - name: testingpython36
     service: testingpython36
-    command: coverage run -m pytest
+    command: coverage run -m pytest --ignore=src/tests/functional
   - name: testingpython37
     service: testingpython37
-    command: coverage run -m pytest
+    command: coverage run -m pytest --ignore=src/tests/functional
   - name: testingpython38
     service: testingpython38
     command: bin/tests_n_reports.sh
+
+- name: functionaltests
```

Comment:
You can put spaces in the names which makes the pipeline steps more readable in the UI https://github.com/carbonblack/developer-network/blob/master/codeship-steps.yml